### ISSUE TITLE
Prefer slightly bolder text in settings and suspended pages

### ISSUE THREAD

**Issue by fwextensions** [deanoemcke/thegreatsuspender#734]
This is personal preference, and depends on the resolution of your display, but the Segoe with `font-weight: 300` looks a bit spindly on my 1900x1200 24" monitor on Win7 (the s's especially):

![image](https://user-images.githubusercontent.com/61631/44623524-9644ad00-a885-11e8-958c-17739afe23a0.png)

To me, removing the `font-weight` style looks better:

![image](https://user-images.githubusercontent.com/61631/44623531-c8560f00-a885-11e8-88dc-3cab469dcd38.png)

**Comment by deanoemcke** [comment]
I've had a similar thought. Although on last check (two days ago) it passed my sniff test.
Will keep this on the table for possible consideration in the future @liamjohnston

**Comment by liamjohnston** [comment]
@fwextensions two things:

1. Could you also provide screenshots of a suspended tab and the options popup? The text in the screenshot is of the settings checkbox options - which is just about the smallest in TGS (14px). So I'm wondering if the solution could be to bump it up instead of changing the weights.

2. Would you mind taking a look at this branch and telling me what you think of the legibility? On the suspended tab, popup and settings section. I'd appreciate it :)
https://github.com/deanoemcke/thegreatsuspender/tree/fix/text-legibility

Basically I removed any instances of less-than-normal font weights. 

Would be good to test this on some other OSes and screens, but I don't have the setup for that :/

The fonts looked great on my screen already (retina MBP, San Fransisco font I think?), but it's not worse with this change I don't think... or if it is, it's negligible.

Thanks!

**Comment by liamjohnston** [comment]
Thanks heaps for the screenshots 👍 I've a changes to only render the light-weight font on high-dpi screens :) It's on the latest `master` now.
